src/renderer/common/hooks/useApi.ts

```diff
@@ -27,7 +27,7 @@ export function useApi<T extends TApiMethodName>(_requestId: string | undefined,
         return () => {
             store.dispatch(apiActions.clean.build(requestId));
         };
-    }, []);
+    }, [requestId, store]);
```

Comment:
seems good

Reply:
the clean dispatch occurs at the unmount of the component with the last requestId

Reply:
yes, `requestId` is already a "first mount" useMemo, and `store` is a useContext "constant" (so to speak) ... so the behaviour with `[]` empty deps array is identical ... looks like a safe refactor that satisfies the compiler / linter

Reply:
note: React strict mode, effects run twice in dev mode